composer.json

```diff
@@ -1,11 +1,15 @@
 {
-    "name": "graylog2/gelf-php",
-    "description": "A php implementation to send log-messages to a GELF compatible backend like Graylog2.",
+    "name": "proximedia/gelf-php",
+    "description": "A php implementation to send log-messages to a GELF compatible backend like Graylog2. Added TCP support.",
```

Comment:
I'd also prefer not changing the description. A better place for such notes is the README.